### PR TITLE
Adds Tator Tots to the game, a new antag type.

### PR DESCRIPTION
**Pull request by Niknakflak**
Not really, it's food.

:cl: NikNak
add: Added tator tots, made my putting a potato in the food processor
tweak: French fries are now made by cutting up a potato into wedges and putting the wedges (plate and all) into the food processor
/:cl:

Sprites were made by me, during class so they are really bad.
Potato wedges on a plate: 
image
(Yes the plate appears magically after you cut the potato, just as plates magically appear when you put stuff in food processors.)

Tator tot
image

I also didn't test this but it compiled :D

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
As by being raised into a family who's only primary sustanance on the bread line is potatoes with a side of mashed potato as big as your head & chips of all sorts i am the processed food potato expert. The wedges are either triangular chips (a bit like a orange segment) or full on triangles with a pulverized middle. I can't tell from the super small sprites but triangle wedges are usually big so i assume its the former.

That does not look like a tator tot, it is a imposterous tator tot wearing a chameleon despite wearing the same body form. Tots are roughly rectagonal with round edges and lava-hot in the middle because of the horrible implications of cooks asking what they can do over whether they should do it leading to over efficient cooking and burnt throats and chests.

**Comment by Jordie0608** [comment]
That's a lot of analysis for what looks like a yellow scribble and an angry rectangle.

**Comment by WJohn** [comment]
Huh. We didn't actually have tator tots in game after all these years until now. Who knew?

I mean I just assumed that we had those, it seemed too logical not to.

**Comment by Niknakflak** [comment]
ur welcome

**Comment by TehZombehz** [comment]
Best NikNak.

**Comment by Dorsisdwarf** [comment]
Is this an american thing. I have no idea what a tator tot is.

**Comment by Niknakflak** [comment]
https://en.wikipedia.org/wiki/Tater_tots

**Comment by Super3222** [comment]
> Is this an american thing. I have no idea what a tator tot is.

they're mini hashbrowns that taste more potato than anything

File: code/modules/hydroponics/grown/potato.dm [hunk 1]
```diff
@@ -50,6 +59,16 @@
 		else
 			user << "<span class='warning'>You need five lengths of cable to make a potato battery!</span>"
 			return
+	if(W.is_sharp())
+		user << "<span class='notice'>You cut the potato into wedges with [W].</span>"
+		var/obj/item/weapon/reagent_containers/food/snacks/grown/potato/wedges/Wedges = new /obj/item/weapon/reagent_containers/food/snacks/grown/potato/wedges
+		if(!remove_item_from_storage(user))
+			user.unEquip(src)
+		user.put_in_hands(Wedges)
```
Comment (Niknakflak) [inline code/modules/hydroponics/grown/potato.dm]: Why
It's exactly the same as the shiv carrot code, which is where it's from.

Comment (AnturK) [inline code/modules/hydroponics/grown/potato.dm]: Ignore, i read this wrong.

**Comment by Crushtoe** [comment]
Is it a datum antag with full modulability and only -50 lines of code added? If not, you're a failure, NikNak.

**Comment by ike709** [comment]
>Is this an american thing. I have no idea what a tator tot is.

This is how you know humanity has failed.

**Pull request merged**